static/app/views/settings/projectUserFeedback/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types/organization';
+import type {Project} from 'sentry/types/project';
+import withOrganization from 'sentry/utils/withOrganization';
+import SettingsPageHeader from 'sentry/views/settings/components/settingsPageHeader';
+import TextBlock from 'sentry/views/settings/components/text/textBlock';
+import PermissionAlert from 'sentry/views/settings/project/permissionAlert';
+
+type RouteParams = {
+  projectId: string;
+};
+type Props = RouteComponentProps<RouteParams, {}> & {
+  organization: Organization;
+  project: Project;
+};
+
+function ProjectUserFeedback({organization, project, params: {projectId}}: Props) {
+  const handleClick = () => {
+    Sentry.showReportDialog({
+      // should never make it to the Sentry API, but just in case, use throwaway id
+      eventId: '00000000000000000000000000000000',
+    });
+  };
+
+  useEffect(() => {
+    window.sentryEmbedCallback = function (embed) {
+      // Mock the embed's submit xhr to always be successful
+      // NOTE: this will not have errors if the form is empty
+      embed.submit = function (_body) {
+        this._submitInProgress = true;
+        window.setTimeout(() => {
+          this._submitInProgress = false;
+          this.onSuccess();
+        }, 500);
+      };
+    };
+
+    return () => {
+      window.sentryEmbedCallback = null;
+    };
+  }, []);
+
+  return (
+    <SentryDocumentTitle title={t('User Feedback')} projectSlug={project.slug}>
+      <SettingsPageHeader
+        title={t('User Feedback')}
+        action={
+          <ButtonList>
+            <LinkButton href="https://docs.sentry.io/product/user-feedback/">
+              {t('Read the Docs')}
+            </LinkButton>
+            <Button priority="primary" onClick={handleClick}>
+              {t('Open the Crash Report Modal')}
+            </Button>
+          </ButtonList>
+        }
+      />
+      <TextBlock>
+        {t(
+          `Don't rely on stack traces and graphs alone to understand
+            the cause and impact of errors. Enable the User Feedback Widget to collect
+            your users' comments at anytime, or enable the Crash Report Modal to collect additional context only when an error occurs.`
+        )}
+      </TextBlock>
+      <PermissionAlert project={project} />
+      <Form
+        saveOnBlur
+        apiMethod="PUT"
+        apiEndpoint={`/projects/${organization.slug}/${projectId}/`}
+        initialData={project.options}
+      >
+        <Access access={['project:write']} project={project}>
+          {({hasAccess}) => <JsonForm disabled={!hasAccess} forms={formGroups} />}
+        </Access>
+      </Form>
+    </SentryDocumentTitle>
+  );
+}
+
+const ButtonList = styled('div')`
+  display: inline-grid;
+  grid-auto-flow: column;
+  gap: ${space(1)};
+`;
+
+export default withOrganization(ProjectUserFeedback);
```

Comment:
is `withOrganization` HoC still the preferred method of getting org from context?

Reply:
not sure, i know we've been getting rid of some of them but i didn't want to mess with it here